shopfloor/actions/data.py

```diff
@@ -43,6 +43,7 @@ def _picking_parser(self):
             "note",
             ("partner_id:partner", self._partner_parser),
             ("carrier_id:carrier", self._simple_record_parser()),
+            ("ship_carrier_id:ship_carrier", self._simple_record_parser()),
```

Comment:
IDT we need another carrier here. The carrier is always for shipping. I would change the `carrier` parser to something like:
```py
[...]
("carrier_id:carrier", self._carrier_parser()),
[...]
def _carrier_parser(self, rec, fname):
    return (rec.carrier_id or rec.ship_carrier_id).jsonify(self._simple_record_parser(), one=True)
```
Maybe @jgrandguillaume could shed some light on this need.